test/fuzzing/request_proposal_fuzz.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ordering_service_fixture.hpp"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, std::size_t size) {
+  static fuzzing::OrderingServiceFixture fixture;
+
+  if (size < 4) {
+    return 0;
+  }
+
+  std::shared_ptr<OnDemandOrderingServiceImpl> ordering_service_;
+  std::shared_ptr<OnDemandOsServerGrpc> server_;
+
+  auto proposal_factory = std::make_unique<MockUnsafeProposalFactory>();
+  ordering_service_ = std::make_shared<OnDemandOrderingServiceImpl>(data[0], std::move(proposal_factory));
```

Comment:
1. `transaction_limit` has `size_t`, so it might be better give at least 2 bytes, instead of one
2. there's `number_of_proposals` that also might be fuzzed
3. might be `initial_round` is also applicable for a fuzzing as well, though not sure

Same for the next test

Reply:
Well, i thought about it. In my opinion the main purpose of fuzzing is to find ways how iroha could be broken via some malicious input. That means a typical ordering gate under attack will probably be initialised with more or less default parameter values.
Feel free to discuss though.